irohad/ametsuchi/impl/flat_file/flat_file.cpp

```diff
@@ -94,23 +95,21 @@ namespace {
 std::unique_ptr<FlatFile> FlatFile::create(const std::string &path) {
```

Comment:
Why uptr?
Should it be optional?

Reply:
It was initially unique_ptr

Reply:
I think, also, we need to rework return value from `create()` with optional type. 

The reason is this file not satisfy our policy about pointers.